config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,7 @@
+Sidekiq.configure_server do |config|
```

Comment:
Do we NEED this file?  Hardcoding to `localhost` seems like it's not going to work in production.